src/components/DropDown/DropDown.tsx

```diff
@@ -0,0 +1,51 @@
+import styles from "./dropdown.module.css";
+import { BsThreeDots } from "react-icons/bs";
+import { useState } from "react";
+import { useClickOutside } from "../../hooks/useClickOutside";
+import { useAppDispatch, useAppSelector } from "../../app/hooks";
+import { Posts } from "../../types/posts.types";
+import {
+  deletePost,
+  setEditPost,
+  setPostModal,
+} from "../../features/postsSlice";
```

Comment:
Suggestion make a jsoncongif for the imports

Reply:
yup, will do!